CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 cmake_minimum_required(VERSION 3.16)
 project(backscrub CXX)
 
+# allow override of Tensorflow location
 if(NOT DEFINED TENSORFLOW)
   set(TENSORFLOW tensorflow)
+  set(TF_CHECKOUT true)
 endif()
 
+# find Git and use it to get backscrub version
 find_package(Git)
 if(GIT_FOUND)
   execute_process(
```

Comment:
IMHO `git describe --long --tags --always --dirty` gives more descriptive results …

Reply:
You may be right, I'm not really fussed.. I tried both on current main and experimental branches:
```bash
% git checkout experimental
...
% git describe --long --always --dirty --all
heads/experimental-0-gaa243fc
% git describe --long --always --dirty --tags
v0.3.0-7-gaa243fc
% git checkout main
...
% git describe --long --always --dirty --all
heads/main-0-g7ec492c
% git describe --long --always --dirty --tags
7ec492c
```
I would note that this is not changed by this PR :smile: